CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+Unreleased changes
+==================
+
+Breaking Changes
```

Comment:
Yeah, it's not an API change, I erred on the side of caution and put it in just in case anyone is depending on them being in that place. wdyt?